README.md

```diff
@@ -1,109 +1,134 @@
-
-## Features
-* Core module
-  * Arena allocator & container
-  * Logger
-  * Types library
-  * Event submodule
-    * Listener system (Java style)
-    * Action delegate (C# style)
-  * Input system
-  * Meta programming module
-* Graphics module
-  * OpenGL abstraction layer
-  * Vulkan abstraction layer (in development)
-  * Shaders
-  * Window
-  * Begui (Dear ImGui abstraction layer)
-## Build
-For now there is two methods how you can use bebone is your project(build locally).
-1. **Build manually**<br>
-  First of all requirements:
-    - Cmake (At least version 3.21)
-    - Ninja (At least version 1.11.1)
-    - C++ compiler (Have test with Clang 15.0.5 and GCC 12.2.0) 
-    - *Vulkan SDK (if you want to build gfx module)*
-  
-    Firstly lets clone bebone locally(note that you also need to clone all bebone git submodules).
-
-    Secondly lets configure our Cmake configuration with
-    ```bash
-    cmake -B build -G Ninja
-    ```
-
-    Finally you can simply build project with cmake 
-    ```bash
-    cmake --build build
-    ```
-
-2. **Using docker**<br>
-    Currently the docker image is not available, but we are planning to release it towards bebone 0.2v.
-
-Note that bebone is only a **set** of useful libraries/modules, some modules can be used separately.
-
-## Testing
-The testing process is similar to the **Building**. After a successful build, all you have to do is run CTest from ```build\tests\unit``` directory.
-
-## Dependencies
-For convenience, we try to reduce the use of third-party libraries, as it complicates code management and readability.
-At the moment, bebone is only dependent on these libraries:
-  - [Glad](https://github.com/Maksasj/glad/tree/all-extensions) *(opengl function header)*
-  - [Vulkan-Headers](https://github.com/Maksasj/Vulkan-Headers/tree/main) *(vulkan function header)*
-  - [GLFW](https://github.com/Maksasj/glfw/tree/master) *(window management library)*
+![GitHub Repo stars](https://img.shields.io/github/stars/Maksasj/bebone)
+![GitHub Actions Workflow Status](https://img.shields.io/github/actions/workflow/status/Maksasj/bebone/clang_build_win.yml)
+![GitHub License](https://img.shields.io/github/license/Maksasj/bebone)
+![Discord](https://img.shields.io/discord/1156584129190711326?logo=discord)
+
+### Links
+1. Source code available at [github.com/Maksasj/bebone](https://github.com/Maksasj/bebone)
+2. Community Discord [discord.gg/bebone](https://discord.gg/v4mcTmuDTb)
+3. **Bebone** documentation available at [github.com/Maksasj/bebone](github.com/Maksasj/bebone/docs/DOCUMENTATION.md)
+
+## Architecture ⚙️
+
+**Bebone** provides multi layer APIs where each level build on top of previous:
+
+- **[Abstraction]()** basic abstraction layer, abstracts most basic things, such as OpenGL or Vulkan
+- **[System]()** advance abstraction layer, implements generalized systems on top of **abstraction layer**, such as **Renderer**

```

Comment:
Change to:
**[System]()** — advance abstraction layer. Implements generalized systems on top of **abstraction layer**, such as **Renderer**